code-challenges.js

```diff
@@ -28,3 +57,26 @@ var hand2 = [5, 5, 3, 3, 4]
 // Expected output: false
 var hand3 = [5, 5, 5, 5, 4]
 // Expected output: false
+
+const isFullHouse = (array) => {
+    let countObject = {}
+    for(let i=0; i < array.length; i++){
+        if (array[i] in countObject){
+            countObject[array[i]]++;
+            
+        }
+        else{
+            countObject[array[i]] = 1;
+        }
+    }
+    if(Object.values(countObject).indexOf(2) > -1 && Object.values(countObject).indexOf(3) > -1){
```

Comment:
I would love for some commenting for this line of code. it took me a while to dissect what your approach was